spring-cloud-alicloud-ans/src/main/java/org/springframework/cloud/alicloud/ans/migrate/MigrateRefreshEventListener.java

```diff
@@ -0,0 +1,99 @@
+package org.springframework.cloud.alicloud.ans.migrate;
+
+import java.util.Iterator;
+import java.util.Set;
+import java.util.concurrent.Executors;
+import java.util.concurrent.TimeUnit;
+
+import javax.annotation.PostConstruct;
+
+import org.apache.commons.logging.Log;
+import org.apache.commons.logging.LogFactory;
+import org.springframework.cloud.context.named.NamedContextFactory;
+import org.springframework.cloud.endpoint.event.RefreshEvent;
+import org.springframework.context.ApplicationListener;
+import org.springframework.core.env.Environment;
+import org.springframework.stereotype.Component;
+
+import com.netflix.loadbalancer.ILoadBalancer;
+
+/**
+ * @author pbting
+ */
+@Component
+public class MigrateRefreshEventListener implements ApplicationListener<RefreshEvent> {
+	private final static Log log = LogFactory.getLog(MigrateRefreshEventListener.class);
+
+	private final static int CHECK_INTERVAL = 1;
+
+	private final static String MIGRATE_SWITCH = "sca.migrate.ans.switch";
```

Comment:
just a suggestion for use full name like `spring.cloud.alicloud.migarte.ans.switch`

Reply:
ok,good suggestion